src/test/resources/codegeneration/imperative/iaf_psc_alpha_imperative.nestml

```diff
@@ -14,7 +14,7 @@ neuron iaf_psc_alpha_mc_imperative:
     # Membrane time constant in ms
     Tau ms = 10
     # Membrane capacitance in pF
-    C pF = 250
+    Cap pF = 250
```

Comment:
Is this because of the symbol for Coulomb?
I think it would be better to replace this by ``C_m``